File: security/src/main/java/top/bettercode/summer/security/token/Token.java
package top.bettercode.summer.security.token;

import java.time.Instant;

public class Token extends InstantAt {

  private static final long serialVersionUID = 1L;

  private final String tokenValue;

  public Token(String tokenValue, Instant issuedAt, Instant expiresAt) {
    super(issuedAt, expiresAt);
    this.tokenValue = tokenValue;
  }


  public String getTokenValue() {
    return tokenValue;
  }

}

File: security/src/main/kotlin/top/bettercode/summer/security/token/Token.kt
package top.bettercode.summer.security.token

import java.time.Instant

class Token(val tokenValue: String, issuedAt: Instant?, expiresAt: Instant?) : InstantAt(issuedAt, expiresAt){
    companion object {
        @Suppress("ConstPropertyName")
        private const val serialVersionUID = 1L
    }

}
